source/install/prepare-mattermost-database.rst

```diff
@@ -204,3 +207,29 @@ When the PostgreSQL database is installed, and the initial setup is complete, yo
 .. important::
 
   If you are upgrading a major version of Postgres, it is essential that ``ANALYZE VERBOSE`` is run on the database post upgrade. This is necessary to re-populate the ``pg_statistics`` table used to generate optimal query plans. The database performance might suffer if this step is not done.
+
+
+Minimum supported version policy
+---------------------------------
+
+To make planning easier and ensure your Mattermost deployment remains fast and secure, we are introducing a policy for updating the minimum supported version of PostgreSQL. The oldest supported PostgreSQL version Mattermost supports will match the oldest version supported by the PostgreSQL community. This ensures you benefit from the latest features and security updates.
+
+This policy change takes effect from Mattermost v10.6, where the minimum PostgreSQL version required will be PostgreSQL 13. This aligns with the PostgreSQL community's support policy, which provides 5 years of support for each major version.
+
+.. note::
+
+  - Mattermost v10.6 is not an :ref:`Extended Support Release (ESR) <about/release-policy:extended support releases>`. Going forward, this database version support policy will only apply to ESR releases.
+  - When a PostgreSQL version reaches its end of life (EOL), Mattermost will require a newer version starting with the next scheduled ESR release. This means the following future PostgreSQL minimum version increases as follows:
```

Comment:
@sadohert - Is this what you had in mind?

Reply:
I think this works.  Combining these 2 resources:

https://www.postgresql.org/support/versioning/

https://docs.mattermost.com/about/mattermost-server-releases.html

I was imagining a table like:

Mattermost ESR | Release Date | Minimum Postgres Version
-- | -- | --
9.11 | Aug 16,2025 | 12.x
10.5 | Feb 16,2025 | 13.x
10.11 | Aug 16, 2025 | 13.x
11.5 * | Feb 16, 2026 | 14.x

\* Forecasted version and date

Reply:
@sadohert - One concern for the table is that it can lead to further maintenance effort for the team. Because now it's one more item to keep track of and update every ESR. I was thinking that just the 2 lines would be sufficient for customers to get an idea and extrapolate. WDYT?

Reply:
@agnivade - I share the same concerns about maintenance. @sadohert - Are you open to publishing what we have drafted and iterate if we start receiving feedback asking for further clarity?

Reply:
I feel the maintenance effort required to update a single row in the table is justified in this case as it adds valuable clarity for customers and eliminates any potential ambiguity. We could consider incorporating this as part of a playbook's checklist for the ESR release process."

Reply:
@streamer45 - Understood. Please confirm the next 3-4 Mattermost server ESR versions and the associated PG minimum supported version increases please? I'll introduce the table to the docs. I ask for the confirmation because I _think_ we want v10.6 (not an ESR)/PG 13.x, v10.11/PG 14.x, and v11.5/PG 15x, but I could be wrong. Looking to nail this down, then update the docs. Thanks!

Reply:
@cwarnermm Working on top of what @sadohert provided, I think this would be accurate:

Mattermost Version | Release Date | Minimum Postgres Version
-- | -- | --
9.11 ESR | Aug 2025 | 11.x
10.5 ESR | Feb 2025 | 11.x
10.6 | March 2025 | 13.x
10.11 ESR | Aug 2025 | 13.x
11.5 ESR * | Feb 2026 | 14.x

\* Forecasted version and date



@agnivade Please confirm as well.

Reply:
Agreed on maintenance issue... my hope was we could do this:

> consider incorporating this as part of a playbook's checklist for the ESR release process.

Once a year it'll be a `nop` action, and once a year it'll be an update to the table.

Reply:
On 11.5 ESR, we will still be using 13.x. Because 14.x goes EOL on 12 Nov 2026.

Reply:
My bad.  I was using the "Final Release" date on the Postgres docs as the EOL date... 

How do we know the actual EOL?

Reply:
https://endoflife.date/postgresql

Reply:
@sadohert - I agree with a release playbook maintenance step to ensure this guidance remains accurate. @amyblais - How do you recommend we approach this, given that it isn't something we need to review on a monthly basis?

Reply:
I'm still seeing the minimum PG version for `11.5`  as Postgres `14`.

Postgres 13 goes EOL on Nov. 13th/2025, according to https://endoflife.date/postgresql

So this coming November PG 13 goes EOL... I thought our policy was that the next ESR will bump... and thus our minimum goes to 14?

@agnivade ?

Reply:
Oops, my brain stopped functioning there. You are right @sadohert.

Reply:
I'll add this step to the release playbook.

Reply:
Thanks, @amyblais!

@sadohert - I believe this PR is ready for your final review.